main.py

```diff
@@ -110,6 +110,34 @@ def main():
 		mouseBall.setPos(moveData["currentPos"])
 		mouseBall.setVelocity(mouseVelocity)
 
+
+		#gravity adjustment
+		if heldKeys[pygame.K_i]:
+			GRAVITY = max(0.0001,GRAVITY+0.1)
```

Comment:
lWhy 0.001? I think we can allow it to go all the way down to zero gravity ;)

Also, since this is adding gravity, is a minimum needed here? Why not maximum value?